EAD/EADSession3/Servlets/SearchEmployeeServlet.java

```diff
@@ -0,0 +1,72 @@
+package GET2018.EAD.EADSession3.Servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import GET2018.EAD.EADSession3.DAO.ConnectionToDB;
+import GET2018.EAD.EADSession3.DAO.EmployeeQueries;
+
+/**
+ * This Servlet Searches Employee using given FirstName and LastName
+ * 
+ * @author Mohit Sharma
+ *
+ */
+public class SearchEmployeeServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+	private Connection connectionToDB;
+
+	protected void doGet(HttpServletRequest request,
+			HttpServletResponse response) throws ServletException, IOException {
+		try {
+			connectionToDB = ConnectionToDB.getConnection();
+			response.setContentType("text/html");
+			PrintWriter fileWriter = response.getWriter();
+			String firstName = request.getParameter("firstName");
```

Comment:
There are not any null checks in parameters coming from request (Either these should be handled at client side or server side).